### DISCUSSION TITLE
New York Times

### DISCUSSION THREAD

**Discussion by pranavkrishnasuresh** [elrumo/macOS_Big_Sur_icons_replacements#1245] [Icon request]
Can someone please make an icon for New York Times Big Sur style?

The current icon is basically a white circle with the fancy E inside.

**Comment by OMGzilla-git** [comment]
I just submitted one, but until it gets approved, you can get it off my profile.
https://macosicons.com/u/OMGzilla

I saw someone else has one approved already, but mine's a little simpler. Good to have options!

**Reply by pranavkrishnasuresh** [reply]
wow awesome!! thank u so much.

**Reply by pranavkrishnasuresh** [reply]
Could you also make one for Sling TV if possible? if you could center this logo in a white square that would be great!

![](https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Sling_TV_logo.svg/1200px-Sling_TV_logo.svg.png)

**Reply by cake03** [reply]
https://github.com/elrumo/macOS_Big_Sur_icons_replacements/discussions/1247
